mlflow/server/js/src/experiment-tracking/components/traces/TraceDataDrawer.tsx

```diff
@@ -155,9 +155,11 @@ export const TraceDataDrawer = ({
       return (
         <div
           css={{
-            height: `calc(100% - ${theme.spacing.lg}px)`,
+            height: '100%',
             marginLeft: -theme.spacing.lg,
             marginRight: -theme.spacing.lg,
+            marginBottom: -theme.spacing.lg,
+            overflow: 'hidden',
```

Comment:
this is to prevent the drawer scrolling like this:

https://github.com/user-attachments/assets/377b8b6d-b3f2-4f7f-bd62-7c596cabbf3e